test/e2e/sourcery.js

```diff
@@ -1,49 +1,39 @@
-        //Click to expand select user dropdown
-        browser.element('css selector', '#react-select-2--value', function(result) {
-            if(result.status != -1) { 
-                browser.click('#react-select-2--value');
-            }
-        });
-        //Select from expanded droprown
-        browser.element('css selector', '[aria-label="Demo User"]', function(result) {
-            if(result.status != -1) { 
-                browser.click('css selector', '[aria-label="Demo User"]');
-            }
-        });
-        //Assert value is selected
-        browser.assert.containsText('#react-select-2--value-item', 'Demo User');
-        //Click to expand select role dropdown
-        browser.element('css selector', '#react-select-3--value', function(result) {
-            if(result.status != -1) { 
-                browser.click('css selector', '#react-select-3--value');
-            }
-        });
-        //Select from expanded droprown
-        browser.element('css selector', '[aria-label="Admin"]', function(result) {
-            if(result.status != -1) { 
-                browser.click('css selector', '[aria-label="Admin"]');
-            }
-        });
-        //Assert value is selected
-        browser.assert.containsText('#react-select-3--value-item', 'Admin');
-        //Click submit button
-        browser.element('css selector', '[type="submit"]', function(result) {
-            if(result.status != -1) {
-                browser
-                .click('css selector', '[type="submit"]')
-                .waitForElementVisible('.user-info__title');
-            }
-        });
-        //Assert if expected user is logged in
-        browser.assert.containsText('.user-info__title', 'Demo User')
-            .saveScreenshot(conf.imgpath(browser) + 'Demo.png')
-            .end();
+        .waitForElementVisible(common.pageTitle)
+        
+        .click(login.userSelect)
+        .click(login.getSpecificSelectUserOption(userUnderTest))
+        .click(login.roleSelect)
+        .click(login.getSpecificSelectUserOption(roleUnderTest)) 
+        .assert.containsText(login.userSelectedValue, userUnderTest)
+        .click(common.submitButton)
+        .waitForElementVisible(timeLogging.loggedInUsersName)
+        .assert.containsText(timeLogging.loggedInUsersName, userUnderTest)
+        .assert.containsText(menuBar.timeLoggingItem, menuBar.timeLogging)
+        .assert.containsText(menuBar.projectsItem, menuBar.projects)
+        .assert.containsText(menuBar.clientsItem, menuBar.clients)
+        .assert.containsText(menuBar.timeEntriesItem, menuBar.timeEntries)
+        .assert.containsText(menuBar.tasksItem, menuBar.tasks)
+        .assert.containsText(menuBar.invoicesItem, menuBar.invoices)
+        .assert.containsText(menuBar.activeNavigationTab, menuBar.timeLogging)
+        .assert.cssProperty(menuBar.activeNavigationTab, 'color', common.blueColor )
+
+        var d = new Date();
```

Comment:
not a good variable name
also does it make sense to move it up beside other test data?